down.go

```diff
@@ -54,11 +54,15 @@ func DownTo(db *sql.DB, dir string, version int64, opts ...OptionsFunc) error {
 			return err
 		}
 
-		current, err := migrations.Current(currentVersion)
-		if err != nil {
+		if currentVersion == 0 {
 			log.Printf("goose: no migrations to run. current version: %d\n", currentVersion)
 			return nil
 		}
+		current, err := migrations.Current(currentVersion)
+		if err != nil {
+			log.Printf("goose: migration file not found for current version (%d), error: %s,\n", currentVersion, err)
```

Comment:
Small nit, can you remove the comma `,` at the end, otherwise it looks like there is missing output.

```
goose: migration file not found for current version (7), error: no current version found,
goose run: no current version found
```

Thanks for fixing that up. This certainly should have surfaced an error.